force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -31,8 +31,12 @@ public with sharing class VariablesDataTypesOperators {
      * @param b The second number.
      * @return The sum of the two numbers, or null if either number is null.
      */
-    public static Integer addition(Integer a, Integer b) {  
-        return null; // Replace null with the variable you used to store the result
+    public static Integer addition(Integer a, Integer b) { 
+        if (a==null || b== null){
+            return null;
+        } else {
+            return a + b;
+        }   // PASS
```

Comment:
nice job. You honestly dont even need the else statement. 

you could just return a + b;